### ISSUE TITLE
Elixir 1.4.x compile warnings

### ISSUE THREAD

**Issue by developerworks** [falood/exsync#7]
```
Compiling 6 files (.ex)
warning: variable "self" does not exist and is being expanded to "self()", please use parentheses to remove the ambiguity or change the variable name
  lib/exsync.ex:11

warning: variable "beam_dirs" does not exist and is being expanded to "beam_dirs()", please use parentheses to remove the ambiguity or change the variable name
  lib/exsync/config.ex:9

warning: Dict.values/1 is deprecated, use the Map module for working with maps or the Keyword module for working with keyword lists
  lib/exsync/config.ex:24

warning: Dict.take/2 is deprecated, use the Map module for working with maps or the Keyword module for working with keyword lists
  lib/exsync/config.ex:23

warning: variable "app_source_dir" does not exist and is being expanded to "app_source_dir()", please use parentheses to remove the ambiguity or change the variable name
  lib/exsync/config.ex:25

warning: variable "src_dirs" does not exist and is being expanded to "src_dirs()", please use parentheses to remove the ambiguity or change the variable name
  lib/exsync/config.ex:19
```

**Comment by falood** [comment]
I have fixed it at the time elixir v1.4.0 released, but forgot to release new version 😂
Thank you!

**Comment by falood** [comment]
v0.1.4 released
